client/my-sites/plan-features/index.jsx

```diff
@@ -412,10 +418,26 @@ class PlanFeatures extends Component {
 				features,
 				planName
 			} = properties;
+			
+			let planFeatures = features;
```

Comment:
Chatted with @mendezcode and realized my original comment here was all wet. This code was safe because he never actually modifies `planFeatures`, he only reassigns it. My apologies. Gonna send me back to js school. I would however suggest a more descriptive variable name to the purpose of what we're doing here rather than `planFeatures`. Maybe something like `abTestVariantFeatures`.